# ðŸ“š Bibliography

### Python Libraries
- Cock, P. J. A., et al. (2009). **Biopython: freely available Python tools for computational molecular biology and bioinformatics**. *Bioinformatics, 25*(11), 1422â€“1423.  
- Pedregosa, F., et al. (2011). **Scikit-learn: Machine Learning in Python**. *Journal of Machine Learning Research, 12*, 2825â€“2830.  
- Hunter, J. D. (2007). **Matplotlib: A 2D Graphics Environment**. *Computing in Science & Engineering, 9*(3), 90â€“95.  
- Waskom, M. L. (2021). **Seaborn: statistical data visualization**. *Journal of Open Source Software, 6*(60), 3021.  
- Huerta-Cepas, J., Serra, F., & Bork, P. (2016). **ETE 3: Reconstruction, Analysis, and Visualization of Phylogenomic Data**. *Molecular Biology and Evolution, 33*(6), 1635â€“1638.  

### External Tools
- Nawrocki, E. P., & Eddy, S. R. (2013). **Infernal 1.1: 100-fold faster RNA homology searches**. *Bioinformatics, 29*(22), 2933â€“2935.  
- Price, M. N., Dehal, P. S., & Arkin, A. P. (2010). **FastTree 2 â€“ Approximately Maximum-Likelihood Trees for Large Alignments**. *PLoS ONE, 5*(3), e9490.  
- Lorenz, R., et al. (2011). **ViennaRNA Package 2.0**. *Algorithms for Molecular Biology, 6*(1), 26.  

